dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -18,6 +18,29 @@ javax.enterprise.inject.spi.configurator=jakarta.enterprise.inject.spi.configura
 javax.enterprise.inject.spi=jakarta.enterprise.inject.spi
 javax.enterprise.inject=jakarta.enterprise.inject
 javax.enterprise.util=jakarta.enterprise.util
+javax.faces=jakarta.faces
+javax.faces.annotation=jakarta.faces.annotation
+javax.faces.application=jakarta.faces.application
+javax.faces.bean=jakarta.faces.bean
+javax.faces.component=jakarta.faces.component
+javax.faces.component.behavior=jakarta.faces.component.behavior
+javax.faces.component.html=jakarta.faces.component.html
+javax.faces.component.search=jakarta.faces.component.search
+javax.faces.component.visit=jakarta.faces.component.visit
+javax.faces.context=jakarta.faces.context
+javax.faces.convert=jakarta.faces.convert
+javax.faces.el=jakarta.faces.el
+javax.faces.event=jakarta.faces.event
+javax.faces.flow=jakarta.faces.flow
+javax.faces.flow.builder=jakarta.faces.flow.builder
+javax.faces.lifecycle=jakarta.faces.lifecycle
+javax.faces.model=jakarta.faces.model
+javax.faces.push=jakarta.faces.push
+javax.faces.render=jakarta.faces.render
+javax.faces.validator=jakarta.faces.validator
+javax.faces.view=jakarta.faces.view
+javax.faces.view.facelets=jakarta.faces.view.facelets
+javax.faces.webapp=jakarta.faces.webapp
```

Comment:
Where are these actually used? is it for the shared bundle?

Reply:
I used them all as catch all as we tried to get the PR to beta. However, since we have time now, I can filter out which ones are necessary.

Reply:
I believe we only need subset: 
 
**com.ibm.ws.cdi.jsf** only needs: 
jakarta.faces.application;version="[3.0,4.0)",
jakarta.faces.context;version="[3.0,4.0)",

**com.ibm.ws.jsf.shared** only needs 
jakarta.faces;version="[3.0,4.0)",
jakarta.faces.application;version="[3.0,4.0)",
jakarta.faces.context;version="[3.0,4.0)",

Reply:
Additionally, I'll need to review test applications, too

Reply:
These should also be transformed since they're used in **com.ibm.ws.jsf.2.3_fat**:
```
javax.faces;version="[2.3,3)",
javax.faces.annotation;version="[2.3,3)",
javax.faces.application;version="[2.3,3)",
javax.faces.bean;version="[2.3,3)",
javax.faces.component;version="[2.3,3)",
javax.faces.component.behavior;version="[2.3,3)",
javax.faces.component.html;version="[2.3,3)",
javax.faces.component.search;version="[2.3,3)",
javax.faces.component.visit;version="[2.3,3)",
javax.faces.context;version="[2.3,3)",
javax.faces.convert;version="[2.3,3)",
javax.faces.event;version="[2.3,3)",
javax.faces.flow;version="[2.3,3)",
javax.faces.lifecycle;version="[2.3,3)",
javax.faces.model;version="[2.3,3)",
javax.faces.push;version="[2.3,3)",
javax.faces.render;version="[2.3,3)",
javax.faces.validator;version="[2.3,3)",
javax.faces.view;version="[2.3,3)",
javax.faces.view.facelets;version="[2.3,3)",
```

Reply:
The only transformations not necessary, I believe, are: 
javax.faces.webapp=jakarta.faces.webapp
javax.faces.flow.builder=jakarta.faces.flow.builder
javax.faces.el=jakarta.faces.el

I feel like it's just better to include all of them and avoid skipping a couple.